backend/src/BuiltinCli/Libs/Execution.fs

```diff
@@ -75,6 +75,27 @@ let fns : List<BuiltInFn> =
         | _ -> incorrectArgs ())
       sqlSpec = NotQueryable
       previewable = Impure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "cliGetOS" 0
+      description = "Returns the operating system name (e.g. Windows, OSX, Linux)"
+      typeParams = []
+      parameters = [ Param.make "unit" TUnit "" ]
+      returnType = TString
+      fn =
+        (function
+        | _, _, [ DUnit ] ->
+          let os =
+            if RuntimeInformation.IsOSPlatform OSPlatform.Windows then "Windows"
+            else if RuntimeInformation.IsOSPlatform OSPlatform.Linux then "Linux"
+            else if RuntimeInformation.IsOSPlatform OSPlatform.OSX then "OSX"
+            else "Unknown"
+
+          DString os |> Ply
```

Comment:
I think it'd be nicer to return a DU with this information. We have the OS type, so probably an Option or Result of that?